tsconfig.cjs.json

```diff
@@ -2,8 +2,6 @@
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
     "outDir": "./build/cjs",
-    "module": "CommonJS",
-    "target": "ES2017"
```

Comment:
The `target` is already set in the parent tsconfig, so there is no need to set it here again.